### PR TITLE
surface

### PR DESCRIPTION
**Pull request by jdamiba**

### CODE REVIEW

File: r/2015-07-30-3d-surface-plots.Rmd [hunk 1]
```diff
@@ -12,10 +12,6 @@ page_type: example_index
 permalink: r/3d-surface-plots/
 thumbnail: thumbnail/3d-surface.jpg
 ---
-
-```{r, echo = FALSE, message=FALSE}
```
Comment (nicolaskruchten) [inline r/2015-07-30-3d-surface-plots.Rmd]: I'm pretty sure we like this block here... it's in all the files in this repo. Could we add it back in all the open PRs please? :)

**Comment by jdamiba** [comment]
This PR replaces https://github.com/plotly/documentation/pull/1612

**Comment by nicolaskruchten** [comment]
@Mahdis-z can you get this PR updated for inclusion into this repo? basically without the `api_create` calls and following the same pattern as the other files :)

**Comment by nicolaskruchten** [comment]
ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!

@Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

**Comment by Mahdis-z** [comment]
> ah, doh, I see that @jdamiba has updated the PR already for the new pattern! nice job Joe!
> 
> @Mahdis-z then please review this PR to make sure it matches your intention from the previous one it replaces :)

Thanks @jdamiba , all's good

**Pull request merged**